servicebus/data-plane/azure-servicebus/src/main/java/com/microsoft/azure/servicebus/primitives/CoreMessageSender.java

```diff
@@ -901,7 +902,8 @@ public int getPriority()
         }
     }
 
-    private static class DeliveryTagComparator implements Comparator<WeightedDeliveryTag> {
+    private static class DeliveryTagComparator implements Comparator<WeightedDeliveryTag>, Serializable {
```

Comment:
[SpotBugs-P2]: 
com.microsoft.azure.servicebus.primitives.CoreMessageSender$DeliveryTagComparator implements Comparator but not Serializable